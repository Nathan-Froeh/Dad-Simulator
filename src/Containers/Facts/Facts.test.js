import React from 'react';
import {
  Facts,
  mapDispatchToProps,
  mapStateToProps
} from './Facts';
import {shallow} from 'enzyme';


describe('Facts', () => {
  let wrapper
  const props = {
    getRandomFacts: jest.fn(),
    randomFact: 'randomFact'
  }
  beforeEach(() => {
    wrapper = shallow(<Facts {...props}/>)
  })

  it('Fatcs should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })


  describe('mapDispatchToProps', () => {

  })  

  describe('mapStateToProps', () => {

  })
})