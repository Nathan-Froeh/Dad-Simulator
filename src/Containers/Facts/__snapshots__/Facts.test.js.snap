// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Facts Fatcs should match snapshot 1`] = `ShallowWrapper {}`;
